hypha/apply/review/templates/review/review_edit_form.html

```diff
@@ -13,7 +13,7 @@
     {% include "forms/includes/form_errors.html" with form=form %}
 
     <div class="wrapper wrapper--medium wrapper--inner-space-medium">
-        <form class="form form--with-p-tags form--scoreable" action="" method="post" novalidate>
+        <form id="review-form-edit" class="form form--with-p-tags form--scoreable" action="" method="post" novalidate>
```

Comment:
This id needs to be added to `hypha/apply/review/templates/review/review_form.html` as well.

Reply:
Whoops, nice catch, added!